src/react/components/pages/projectSettings/projectMetrics.tsx

```diff
@@ -31,16 +36,32 @@ export interface IProjectMetricsState {
 export default class ProjectMetrics extends React.Component<IProjectMetricsProps, IProjectMetricsState> {
     public state = {
         loading: true,
+        hoveredCell: null,
         sourceAssets: [],
         projectAssetsMetadata: [],
     };
 
+    private tipStyle = {
```

Comment:
can the inline styles live in a `scss` file or typical for this component to use them like this?

Reply:
Agreed - these should be moved to SCSS file.